base/applications/cacls/lang/id-ID.rc

```diff
@@ -41,32 +41,32 @@ Singkatan:\n\
     IDS_ABBR_CHANGE "C"
     IDS_ABBR_NONE "N"
     IDS_ALLOW ""
-    IDS_DENY "(DENY)"
+    IDS_DENY "(MENOLAK)"
     IDS_SPECIAL_ACCESS "(akses spesial:)"
-    IDS_GENERIC_READ "GENERIC_READ"
-    IDS_GENERIC_WRITE "GENERIC_WRITE"
-    IDS_GENERIC_EXECUTE "GENERIC_EXECUTE"
-    IDS_GENERIC_ALL "GENERIC_ALL"
-    IDS_FILE_GENERIC_EXECUTE "FILE_GENERIC_EXECUTE"
-    IDS_FILE_GENERIC_READ "FILE_GENERIC_READ"
-    IDS_FILE_GENERIC_WRITE "FILE_GENERIC_WRITE"
-    IDS_FILE_READ_DATA "FILE_READ_DATA"
-    IDS_FILE_WRITE_DATA "FILE_WRITE_DATA"
-    IDS_FILE_APPEND_DATA "FILE_APPEND_DATA"
-    IDS_FILE_READ_EA "FILE_READ_EA"
-    IDS_FILE_WRITE_EA "FILE_WRITE_EA"
-    IDS_FILE_EXECUTE "FILE_EXECUTE"
-    IDS_FILE_DELETE_CHILD "FILE_DELETE_CHILD"
-    IDS_FILE_READ_ATTRIBUTES "FILE_READ_ATTRIBUTES"
-    IDS_FILE_WRITE_ATTRIBUTES "FILE_WRITE_ATTRIBUTES"
-    IDS_MAXIMUM_ALLOWED "MAXIMUM_ALLOWED"
-    IDS_ACCESS_SYSTEM_SECURITY "ACCESS_SYSTEM_SECURITY"
-    IDS_SPECIFIC_RIGHTS_ALL "SPECIFIC_RIGHTS_ALL"
-    IDS_STANDARD_RIGHTS_REQUIRED "STANDARD_RIGHTS_REQUIRED"
+    IDS_GENERIC_READ "BACAAN-UMUM"
```

Comment:
The names of the file access flags must NOT be translated!!

Reply:
I think these strings should not be in lang resources, since they are not supposed to be translated.

Reply:
Yeah makes no sense to have strings in language-specific files, if they are not translatable at all. Why are they even resource strings?

Reply:
The thing is, every (or most?) language-specific file comes with these strings and that'd be a big pain to review every resource file that contain them. Perhaps this thing should be discussed in the upcoming meeting minute so we can avoid situations in the future like above. Any ideas guys?